src/shared/components/mutationTable/column/ascnMethod/ASCNMethodColumnFormatter.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import {Mutation} from "shared/api/generated/CBioPortalAPI";
+import {hasASCNProperty} from "shared/lib/MutationUtils";
+import SampleManager from "pages/patientView/SampleManager";
+
+/**
+ * @author Avery Wang
+ */
+
+function getASCNMethodValue(mutation: Mutation): string {
+    return hasASCNProperty(mutation, "ascnMethod") ? mutation.alleleSpecificCopyNumber.ascnMethod : "";
+}
+
+export const getDefaultASCNMethodColumnDefinition = () => {
+    return {
+        name: "ASCN Method",
+        tooltip: (<span>Allele Specific Copy Number Method</span>),
+        render: (d: Mutation[]) => ASCNMethodColumnFormatter.renderFunction(d),
+        sortBy: (d: Mutation[]) => getASCNMethodValue(d[0]),
+        download: (d: Mutation[]) => ASCNMethodColumnFormatter.getASCNMethodDownload(d)
+    }
+};
+
+
+export default class ASCNMethodColumnFormatter {
```

Comment:
If this formatter is not reused in some other components, you could just move the class and put the functions inside of ColumnDefinition directly. And I realize the few formatters are actually not used, only the ColumnDefinition. Maybe just move them to a util class? So you can get rid of some files.

Reply:
@onursumer can you clarify this? is @zhx828 suggesting removing this entire file (ASCNMethodColumnFormatter.tsx) and just moving the column definition (name, tooltip, render...) back into MutationTable.tsx?

Reply:
Seems like the actual class `ASCNMethodColumnFormatter` itself has not been referenced anywhere else. So you can just get rid of the class and move the methods inside `getDefaultASCNMethodColumnDefinition`. See the suggestions above.

I don't think @zhx828 is not suggesting moving column definitions back into `MutationTable.tsx`, but maybe group similar group definitions like this into a single utility file. Though, I think it is not a big deal if we keep them in separate files.